jib-core/src/test/java/com/google/cloud/tools/jib/api/ContainerizerTest.java

```diff
@@ -121,4 +128,15 @@ public void testGetImageConfiguration_tarImage() throws InvalidImageReferenceExc
     Assert.assertEquals("tar/image", imageConfiguration.getImage().toString());
     Assert.assertEquals(0, imageConfiguration.getCredentialRetrievers().size());
   }
+
+  @Test
+  public void testGetApplicationLayersCacheDirectory_defaults()
+      throws InvalidImageReferenceException, CacheDirectoryCreationException, IOException {
+    Containerizer containerizer = Containerizer.to(RegistryImage.named("registry/image"));
+    Path applicationLayersCache = containerizer.getApplicationLayersCacheDirectory();
+    Path expectedCacheDir =
+        Paths.get(System.getProperty("java.io.tmpdir"))
+            .resolve(Containerizer.DEFAULT_APPLICATION_CACHE_DIRECTORY_NAME);
```

Comment:
How about verifying the name in the test?
```suggestion
            .resolve("jib-core-application-layers-cache");
```

Reply:
oh right.